README.md

```diff
@@ -33,6 +33,20 @@ Help us maintain an inclusive environment by following our [code of conduct](COD
 
 # About DevTools
 
+## Firefox Advantages
+
+The browser made for developers
+All the latest developer tools in beta, plus experimental features like the Multi-line Console Editor and WebSocket Inspector.
+
+* [Inactive CSS](https://hacks.mozilla.org/2019/10/firefox-70-a-bountiful-release-for-all/#developertools)
+Firefox DevTools now grays out CSS declarations that don’t have an effect on the page. When you hover over the info icon, you’ll see a useful message about why the CSS is not being applied, including a hint about how to fix the problem.
+* [Firefox DevTools](https://mozilladevelopers.github.io/playground/debugger/)
+The new Firefox DevTools are powerful, flexible, and best of all, hackable. This includes a best-in-class JavaScript debugger, which can target multiple browsers and is built in React and Redux
```

Comment:
I think we can remove this point since it doesn't really talk about an actual feature. I'll have to think about what we can do to highlight specialties of our Debugger.